### ISSUE TITLE
0908 not recognized in Nigeria

### ISSUE THREAD

**Issue by pblv** [googlei18n/libphonenumber#863]
Numbers can start with 0908 from Etisalat

Please see http://www.ncc.gov.ng/index.php/index.php?option=com_content&view=article&id=113&Itemid=102

It's currently not accepted.

**Comment by rasmushansen** [comment]
It seems like the issue is the last clause in recognizing Nigerian mobile numbers:
```javascript
(?:1(?:7[34]\d|8(?:04|[124579]\d|8[0-3])|95\d)|287[0-7]|3(?:18[1-8]|88[0-7]|9(?:8[5-9]|6[1-5]))|4(?:28[0-2]|6(?:7[1-9]|8[02-47])|88[0-2])|5(?:2(?:7[7-9]|8\d)|38[1-79]|48[0-7]|68[4-7])|6(?:2(?:7[7-9]|8\d)|4(?:3[7-9]|[68][129]|7[04-69]|9[1-8])|58[0-2]|98[7-9])|7(?:38[0-7]|69[1-8]|78[2-4])|8(?:28[3-9]|38[0-2]|4(?:2[12]|3[147-9]|5[346]|7[4-9]|8[014-689]|90)|58[1-8]|78[2-9]|88[5-7])|98[07]\d)\d{4}|(?:70(?:[13-9]\d|2[1-9])|8(?:0[2-9]|1\d)\d|90[235->missing 8 here<-9]\d)\d{6}
```

**Comment by andyst** [comment]
Thanks for the report including the government doc as evidence!
We've added the issue to our internal queue.

**Comment by penmetsaa** [comment]
@pblv,
Fixed in release 7.2.4: https://libphonenumber.appspot.com/phonenumberparser?number=07020120000&country=NG

**Comment by rasmushansen** [comment]
Thanks!
